cmd/downloader/downloader/downloader_grpc_server.go

```diff
@@ -40,6 +40,11 @@ func (s *GrpcServer) Download(ctx context.Context, request *proto_downloader.Dow
 			}
 		}
 
+		if it.TorrentHash == nil {
```

Comment:
1. We can’t just ignore it, because we need seed this files for other peers
2. It happening because I moved AddSegment after Proto2InfoHas. Let’s try call Proto2InfoHash only if TorrentHash is not nil.